app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -192,6 +197,12 @@ export default {
         },
       });
     },
+    // Custom Event triggered from buttons with Action type "Event"
+    onCustomEvent(event) {
+      alert(
+        `Custom button events not supported yet. Event Type: ${event.type}`
+      );
```

Comment:
Should we dispatch this to the vuex notification stack instead of an alert for the moment?

Reply:
Nah this is hardcore "don't use" debugging at the moment